cli/search/bundle.go

```diff
@@ -0,0 +1,32 @@
+package search
+
+import (
+	"github.com/tarantool/tt/cli/version"
+)
+
+// BundleInfo is a structure that contains specific information about SDK bundle.
+type BundleInfo struct {
+	Version version.Version
+	Prefix  string
```

Comment:
Pls, add a comment about this field. And to make it perfect add a comment about `'Version`.

Reply:
Added.